app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.spec.tsx

```diff
@@ -17,7 +17,6 @@ describe('DocumentCaptureTroubleshootingOptions', () => {
   const serviceProviderContext: ServiceProviderContext = {
     name: 'Example SP',
     failureToProofURL: 'http://example.test/url/to/failure-to-proof',
-    isLivenessRequired: false,
```

Comment:
There's still a few lingering references to this in `spec/javascripts/packages/document-capture/components/document-capture-spec.jsx`.